viz_scripts/scaffolding.py

```diff
@@ -134,16 +137,25 @@ def load_viz_notebook_data(year, month, program, study_type, dynamic_labels, dic
         if (len(dynamic_labels)):
             dic_mode_mapping = mapping_labels(dynamic_labels, "MODE")
             expanded_ct['Mode_confirm'] = expanded_ct['mode_confirm'].map(dic_mode_mapping)
+            labels = dynamic_labels
         else:
             expanded_ct['Mode_confirm'] = expanded_ct['mode_confirm'].map(dic_re)
+            labels = await read_json_resource("label-options.default.json")
+        # If the 'mode_confirm' is not available as the list of keys in the dynamic_labels or label_options.default.json, then, we should transform it as 'other'
+        mode_values = [item['value'] for item in labels['MODE']]
+        expanded_ct['merge_mode_confirm'] = expanded_ct['mode_confirm'].apply(lambda mode: 'other' if mode not in mode_values else mode)
```

Comment:
We already had mode_confirm column. We are merging the rows which had internal labels which aren't present in the label-options's MODE keys into `other`.

Reply:
Would something like `grouped_XX` or `processed_XX` be any more clear?

Reply:
@Abby-Wheelis I would rename it to `mode_confirm_w_other` as discussed?

Reply:
Just to clarify, if I say that something is a `nit`, it doesn't need to hold up the merge. https://en.wikipedia.org/wiki/Nitpicking I would suggest focusing on the important issues - e.g.

> The multiple retrievals of the default JSON file should be fixed before merge; the others can be cleaned up later.

and clean the others up later.

This will ensure that we don't make big changes, which have the potential to break code, while making progress towards the merge.